addons/web/static/tests/core/utils/functions.test.js

```diff
@@ -0,0 +1,56 @@
+/** @odoo-module */
+
+import { memoize, uniqueId } from "@web/core/utils/functions";
```

Comment:
move this below, with a blank line to separate hoot and helpers imports from @web imports, as they come from two different worlds